apache/couchdb-fauxton
# Fix/database creation error feedback

## Overview

When the user attempted to create an invalid database, no feedback was returned. 

I fixed the js error in order to have a notification box with the error message.

## Testing recommendations

Nightwatch test or:

1. Login
2. click on create database
3. Fill an invalid name : ####
4. Click create database
5. It should show a notification box with the error message.

## GitHub issue number

#1009

## Checklist
